src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import './MovieCard.scss';
 
 type Props = {
-  movie: Movie;
+  movie: Movie | null;
```

Comment:
it can't be null, because you have checked it earlier 
<img width="304" alt="image" src="https://user-images.githubusercontent.com/64437769/160365065-d333d3d5-6b8c-4e41-9014-7c5000b6c018.png">